server.py

```diff
@@ -66,11 +65,13 @@ def db_fetch(key):
   else:
     return None
 
-def db_delete(key):
+def db_delete(key: str) -> bool:
   conn = get_db()
   c = conn.cursor()
   c.execute('DELETE FROM store WHERE key = ?', (key,))
+  did_delete = c.rowcount > 0
   conn.commit()
+  return did_delete
```

Comment:
Can we make this a JSON object? e.g. `{"key": "whatever", "deleted": true}`

Reply:
done :)